### ISSUE TITLE
An error while running craft_llm_dataset.py

### ISSUE THREAD

**Issue by Rhymeheart** [GuanlinLee/ART#5]
When I run the code file "craft_llm_dataset.py", the program gets stuck in the function "build_llava" and keeps warning "The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.Setting `pad_token_id` to `eos_token_id`:2 for open-end generation." 

After I set the pad_token_id, the warning disappeared, but the program was still in stuck the  function "build_llava"

But when I comment out the code for the if __name__ == '__main__' module, it loads fine.Could you tell me how to fix this ?
![image](https://github.com/user-attachments/assets/2a2fa251-4749-4aac-bb8b-e729fb3859c5)

**Comment by GuanlinLee** [comment]
In build_llava, we simply call the function from LLAVA. So, if you meet any problem in this part, please open an issue in the corresponding repo.
